atoma-event-subscribe/sui/src/subscriber.rs

```diff
@@ -213,7 +220,24 @@ impl SuiSubscriber {
     #[instrument(skip(self, event_data))]
     async fn handle_prompt_event(&self, event_data: Value) -> Result<(), SuiSubscriberError> {
         debug!("event data: {}", event_data);
-        let request = Request::try_from((self.id, event_data))?;
+        let mut request = Request::try_from((self.id, event_data))?;
+        let metadata = AtomaInputMetadata {
+            user_id: request.params().prompt(),
```

Comment:
I don't understand how the `user_id` is the `prompt`, at most it should be its hash..

Reply:
because now the event subscriber expects channel from input manager. Why do we even have standalone binaries for this? We will never run just event subscriber.

Reply:
because I did not modify the contract and I had to put the user id somewhere for the time being. And the prompt is now empty.
I renamed the function getter, but the field is still prompt, because it's json serialized.